src/renderer/features/proxies/lib/worker-utils.ts

```diff
@@ -61,6 +62,10 @@ function isApiConnected(apis: Record<ChainId, ApiPromise>, chainId: ChainId): bo
   return Boolean(api?.isConnected);
 }
 
+function isAvailableProxiedAccount(proxy: NoID<ProxyAccount>): boolean {
```

Comment:
Function compares 2 values, no proxieds, no accounts 🤔 
`isDelayedProxy` seems to be the most obvious name